### ISSUE TITLE
Run multipage a11y assessment tool in HTML

### ISSUE THREAD

**Issue by atsansone** [flutter/website#9535]
Using pa11y, pull a list of accessibility issues in docs.flutter.dev. These should be human-reviewable in HTML.

Use the following command:

```terminal
pa11y-ci --reporter=pa11y-ci-reporter-html --sitemap https://docs.flutter.dev/sitemap.xml
```

**Comment by atsansone** [comment]
Complete: [docs-flutter-dev.zip](https://github.com/flutter/website/files/12920411/docs-flutter-dev.zip)
